trackomatic.js

```diff
@@ -15,6 +15,31 @@
  * limitations under the License.
 */
 
+var BODY = document.getElementsByTagName('body')[0];
```

Comment:
Shouldn't all this be wrapped in a self executing function to scope the variable names and avoid potential conflicts?